core/src/main/java/com/backblaze/b2/client/structures/B2CreatedApplicationKey.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2018, Backblaze Inc. All Rights Reserved.
+ * License https://www.backblaze.com/using_b2_code.html
+ */
+
+package com.backblaze.b2.client.structures;
+
+import com.backblaze.b2.json.B2Json;
+
+import java.util.Objects;
+import java.util.TreeSet;
+
+public class B2CreatedApplicationKey {
```

Comment:
maybe comment that this is what's returned from b2_create_key and that it differs from the other version in that it has the secret (like your nice comment below for toApplicationKey()).